### PR TITLE
Update yugabyte.json

### PR DESCRIPTION
**Pull request by stevebang**
Move internal scraping to Algolia
- Update start_urls to include latest versions
- Update selectors
- Update advanced settings.

### CODE REVIEW

**Comment by stevebang** [comment]
See related ticket ID285816 for context — besides trying to update the configuration, I'm also trying to figure out what is causing indexing issues. I believe this configuration is good, because I see expected changes from this that we were running since June 25th on internal docsearch-scraper installation.

File: configs/yugabyte.json [hunk 1]
```diff
@@ -1,40 +1,35 @@
...
-    "1037222869"
-  ],
-  "nb_hits": 19779
-}
\ No newline at end of file
+    "custom_settings": {
+      "advancedSyntax": true,
+      "separatorsToIndex": "-_"
+    },
+    "selectors_exclude": [],
```
Comment (s-pace) [inline configs/yugabyte.json:33]:
```suggestion
```
You can remove this setting

File: configs/yugabyte.json [hunk 2]
```diff
@@ -1,40 +1,35 @@
...
-    ".article nav"
-  ],
-  "conversation_id": [
-    "1037222869"
-  ],
-  "nb_hits": 19779
-}
\ No newline at end of file
+    "custom_settings": {
+      "advancedSyntax": true,
```
Comment (s-pace) [inline configs/yugabyte.json:30]:
```suggestion
```
This setting is enabled by default

File: configs/yugabyte.json [hunk 3]
```diff
@@ -1,40 +1,35 @@
...
+      "separatorsToIndex": "-_"
+    },
+    "selectors_exclude": [],
+    "nb_hits": 21609
```
Comment (s-pace) [inline configs/yugabyte.json:34]:
```suggestion
  "conversation_id": [
    "1037222869"
  ],
    "nb_hits": 21609
```
Please keep `conversation_id`

**Review by s-pace — CHANGES_REQUESTED** [review]
See inlined comments. Please avoid superfluous code reformatting

**Pull request merged**